receiver/flinkmetricsreceiver/metadata.yaml

```diff
@@ -0,0 +1,522 @@
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flinkmetrics.taskmanager.status.jvm.memory.direct.used:
+    enabled: true
+    description: The amount of memory used by the JVM for the direct buffer pool.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flinkmetrics.jobmanager.status.jvm.memory.direct.total_capacity:
+    enabled: true
+    description: The total capacity of all buffers in the direct buffer pool.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flinkmetrics.taskmanager.status.jvm.memory.direct.total_capacity:
+    enabled: true
+    description: The total capacity of all buffers in the direct buffer pool.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flinkmetrics.jobmanager.status.jvm.memory.mapped.used:
+    enabled: true
+    description: The amount of memory used by the JVM for the mapped buffer pool.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flinkmetrics.taskmanager.status.jvm.memory.mapped.used:
+    enabled: true
+    description: The amount of memory used by the JVM for the mapped buffer pool.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flinkmetrics.jobmanager.status.jvm.memory.mapped.total_capacity:
+    enabled: true
+    description: The number of buffers in the mapped buffer pool.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flinkmetrics.taskmanager.status.jvm.memory.mapped.total_capacity:
+    enabled: true
+    description: The number of buffers in the mapped buffer pool.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flinkmetrics.jobmanager.status.flink.memory.managed.used:
+    enabled: true
+    description: The amount of managed memory currently used.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flinkmetrics.taskmanager.status.flink.memory.managed.used:
+    enabled: true
+    description: The amount of managed memory currently used.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flinkmetrics.jobmanager.status.flink.memory.managed.total:
```

Comment:
`status` was included because it was a part of the original metrics name. I don't think it adds value, so `status` and `flink` used twice in a metric has been removed